src/pydvl/value/least_core/__init__.py

```diff
@@ -4,6 +4,70 @@
 This package holds all routines for the computation of Least Core data values.
 
 Please refer to :ref:`data valuation` for an overview.
+
+In addition to the standard interface via
+:func:`~pydvl.value.least_core.compute_least_core_values`, because computing the
+Least Core values requires the solution of a linear and a quadratic problem
+*after* computing all the utility values, there is the possibility of performing
+each step separately. This is useful when running multiple experiments: use
+:func:`~pydvl.value.least_core.montecarlo.mclc_prepare_problem` to prepare a
+list of problems to solve, then solve them in parallel with
+:func:`~pydvl.value.least_core.common.lc_solve_problems`.
+
 """
+from enum import Enum
+from typing import Optional
+
+from ...utils import Utility
+from .. import ValuationResult
```

Comment:
I find this one much more readable:

```suggestion
from pydvl.value.results import ValuationResult
```